url.bs

```diff
@@ -2858,10 +2866,36 @@ method, when invoked, must run these steps:
  <li><p>Run the <a for=URLSearchParams>update steps</a>.
 </ol>
 
-<p>The
-<dfn method for=URLSearchParams><code>has(<var>name</var>)</code></dfn>
-method, when invoked, must return true if there is a name-value pair whose name is <var>name</var>
-in <a for=URLSearchParams>list</a>, and false otherwise.
+<hr>
+
+<div class=example id=example-searchparams-sort>
+ <p>It can be useful to sort the name-value pairs in a {{SearchParams}} object, in particular to
+ increase cache hits. This can be accomplished simply through invoking the {{SearchParams/sort()}}
+ method:
+
+ <pre><code class=lang-javascript>
+const url = new URL("https://example.org/?q=🏳️‍🌈&amp;key=e1f7bc78");
+url.searchParams.sort();
+url.search; // "?key=e1f7bc78&amp;q=%F0%9F%8F%B3%EF%B8%8F%E2%80%8D%F0%9F%8C%88"</code></pre>
+
+ <p>To avoid altering the original input, e.g., for comparison purposes, construct a new
+ {{SearchParams}} object:
+
+ <pre><code class=lang-javascript>
+const sorted = (new URLSearchParams(url.search)).sort();</code></pre>
```

Comment:
Thanks, I opted for fixing the example. Might change if we fix #90 as requested I suppose.